WooCommerce/Classes/POS/Utils/Color+WooCommercePOS.swift

```diff
@@ -89,23 +98,11 @@ extension Color {
 
     // MARK: - Buttons
 
-    static var posPrimaryButtonBackground: Color {
-        return Color(
-            UIColor(
-                light: .withColorStudio(.wooCommercePurple, shade: .shade50),
-                dark: .withColorStudio(.wooCommercePurple, shade: .shade30)
-            )
-        )
-    }
+    static var posPrimaryButtonBackground: Color = .accent
```

Comment:
For the app theme, we are using `purple_40` in all themes, this is to give a better contrast with the white text (more context here p1734705352565459/1727138732.960149-slack-C07E7D0R761), but for POS, I noticed the button text is black, so for now, it will use `purple_30` in dark theme.

Was the usage of black text here a design decision or just a dev choice? Personally, I think white looks better and gives a better contrast, but that's just a thought.

| Black | White |
| --- | --- |
| ![Simulator Screenshot - iPad Air 13-inch (M2) - 2024-12-23 at 13 19 11](https://github.com/user-attachments/assets/a4e79f65-d3c7-4c31-a22a-6d5403964bfa) | ![Simulator Screenshot - iPad Air 13-inch (M2) - 2024-12-23 at 12 56 17](https://github.com/user-attachments/assets/5e1fda28-62c8-4e8c-96c1-307b566496ee) |